ui/raidboss/data/04-sb/raid/o12n.txt

```diff
@@ -181,8 +183,11 @@ hideall "--sync--"
 
 
 ### Phase 4: Laser Show Forever
-1800.0 "Enrage" GameLog { code: "0044", line: "\<blip\> Warning\. Calculations indicate.*?" } window 1800,0
-
+# https://xivapi.com/InstanceContentTextData/20623
+# https://xivapi.com/InstanceContentTextData/20624
+# en: <blip> Warning. Calculations indicate increased probability of defeat.\n\n<bleep> Error. Further data analysis required.
+# May be either id depending on which is alive.
+1800.0 "Enrage" BattleTalk2 { instanceContentTextId: "50(8F|90)" } window 1800,0
```

Comment:
To save ourselves headache if we ever have to do future conversions, let's expand this regex, as `(508F|5090)`.